recipes/lapack/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# bug 145
+if test `uname` = "Linux"
+then
+  sed -e 's|/CMAKE/|/cmake/|' -i CBLAS/CMakeLists.txt
+fi
+
+# lapack_testing.py does not support py3k, correction sent
+cp ${RECIPE_DIR}/lapack_testing.py .
```

Comment:
@jakirkham, it was a personal mail, I propose to link to the commit once it is published on their new repo